docs/UsersGuide/source/Quickstart.rst

```diff
@@ -1,320 +1,325 @@
 
-Edit the ``config.sh`` file to set the machine you are running on to ``MACHINE``, use an account you can charge for 
-``ACCOUNT``, and set the name of the experiment with ``EXPT_SUBDIR``. If you have access to the NOAA HPSS from the 
-machine you are running on, those changes should be sufficient; however, if that is not the case (for example, 
-on Cheyenne), or if you have pre-staged the initialization data you would like to use, you will also want to set 
-``USE_USER_STAGED_EXTRN_FILES="TRUE"`` and set the paths to the data for ``EXTRN_MDL_SOURCE_BASEDIR_ICS`` and 
-``EXTRN_MDL_SOURCE_BASEDIR_LBCS``. 
+The command above also binds the local directory to the container so that data can be shared between them. On Level 1 systems, ``<local_base_dir>`` is usually the topmost directory (e.g., /lustre, /contrib, /work, or /home). Additional directories can be bound by adding another ``--bind /<local_base_dir>:/<container_dir>`` argument before the name of the container. 
 
-.. note::
+.. attention::
+   * When binding two directories, they must have the same name. It may be necessary to ``cd`` into the container and create an appropriately named directory in the container using the ``mkdir`` command if one is not already there. 
+   * Be sure to bind the directory that contains the experiment data. 
 
-   If you set up the build environment with the GNU compiler in :numref:`Section %s <SetUpBuild>`, you will
-   have to add the line ``COMPILER="gnu"`` to the ``config.sh`` file.
- 
-At a minimum, the following parameters should be set for the machine you are using:
 
-For Cheyenne:
+.. _SetUpBuildC:
+
+Set up the Build Environment
+============================
+
+Set the build environments and modules within the ``ufs-srweather-app`` directory as follows:
 
 .. code-block:: console
 
-   MACHINE="cheyenne"
-   ACCOUNT="my_account"
-   EXPT_SUBDIR="my_expt_name"
-   USE_USER_STAGED_EXTRN_FILES="TRUE"
-   EXTRN_MDL_SOURCE_BASEDIR_ICS="/glade/p/ral/jntp/UFS_SRW_app/model_data/FV3GFS"
-   EXTRN_MDL_SOURCE_BASEDIR_LBCS="/glade/p/ral/jntp/UFS_SRW_app/model_data/FV3GFS"
+   cd ubuntu20.04-epic-srwapp-1.0/opt/ufs-srweather-app/
+   ln -s /usr/bin/python3 /usr/bin/python
+   source /usr/share/lmod/6.6/init/profile
+   module use /opt/hpc-modules/modulefiles/stack
+   module load hpc hpc-gnu hpc-openmpi hpc-python
+   module load netcdf hdf5 bacio sfcio sigio nemsio w3emc esmf fms crtm g2 png zlib g2tmpl ip sp w3nco cmake gfsio wgrib2 upp
 
-For Hera:
 
-.. code-block:: console
 
-   MACHINE="hera"
-   ACCOUNT="my_account"
-   EXPT_SUBDIR="my_expt_name"
+Build the Executables
+======================
 
-For Jet:
+From the ``ufs-srweather-app`` directory, ``cd`` into the build directory and run the script that builds the SRW App: 
 
 .. code-block:: console
 
-   MACHINE="jet"
-   ACCOUNT="my_account"
-   EXPT_SUBDIR="my_expt_name"
+   cd build
+   source build-srw.sh
 
-For Orion:
+Download and Stage the Data
```

Comment:
Np! NOAA Cloud is Level 1. The data is too big to fit in the container, so it's not included. In the container process, users bind the container to the rest of the system so that they can access the system's data. So users can just use the data that's on the platform they're using if it's a Level 1 or 2 system. I just tried to clarify that in the docs.